compiler/rustc_codegen_llvm/src/debuginfo/metadata.rs

```diff
@@ -993,9 +993,15 @@ pub fn compile_unit_metadata(
     let producer = format!("clang LLVM ({})", rustc_producer);
 
     let name_in_debuginfo = name_in_debuginfo.to_string_lossy();
-    let work_dir = tcx.sess.working_dir.0.to_string_lossy();
```

Comment:
This was necessary because `llvm-dwp` concatenates `DW_AT_comp_dir` to `DW_AT_GNU_dwo_name` - but I'm not sure if it would have unintended consequences.